saleor/attribute/__init__.py

```diff
@@ -38,6 +38,11 @@ class AttributeInputType:
         NUMERIC,
     ]
 
+    # list of the translatable attributes
```

Comment:
Dropdown and mulitselects are also translatable, but we don't include them here. It would be good to mention that:

```suggestion
    # list of the translatable attributes, excluding attributes with choices
```